javers-persistence-sql/src/main/java/org/javers/repository/sql/pico/SchemaManagementDisabledModule.java

```diff
@@ -0,0 +1,40 @@
+package org.javers.repository.sql.pico;
+
+import org.javers.core.pico.JaversModule;
+import org.javers.repository.sql.EmptySchemaManager;
+import org.javers.repository.sql.JaversSqlRepository;
+import org.javers.repository.sql.finders.CdoSnapshotFinder;
+import org.javers.repository.sql.finders.CommitPropertyFinder;
+import org.javers.repository.sql.repositories.CdoSnapshotRepository;
+import org.javers.repository.sql.repositories.CommitMetadataRepository;
+import org.javers.repository.sql.repositories.GlobalIdRepository;
+import org.javers.repository.sql.schema.FixedSchemaFactory;
+import org.javers.repository.sql.schema.JaversSchemaManager;
+import org.javers.repository.sql.schema.TableNameProvider;
+import org.polyjdbc.core.query.QueryRunnerFactory;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+/**
+ * Created by antonio on 13/02/2018.
+ */
+public class SchemaManagementDisabledModule implements JaversModule {
```

Comment:
you don't need to copy-paste the whole class, one `if` in `JaversSqlRepository` would do the job:


```
    @Override
    public void ensureSchema() {
        if (... check the new flag) {
            schemaManager.ensureSchema();
        }
    }
```

Reply:
At the end, please squash your commits

Reply:
you told me to implement
 - SqlRepositoryBuilder.withSchemaManagementEnabled(boolean)
Do you mean that I have to pass this data to JaversSqlRepository?